site/src/components/card.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Card
+description: The card component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D2719%253A10497" allowfullscreen
+---
+
+# Card
+
+The card component is useful for representing data and actions associated with a singular entity within a particular content model. A card can represent an article, page, module, learning path, course, certification, or nearly anything else. The component itself can be used in simple or complex ways. It can be used in a highly prescriptive manner or only as a freeform bucket within which other elements can be placed. Because of its many ins-and-outs, several optional subcomponents, and general flexibility, card can yield many different visual outcomes.
+
+This page discusses the component and subcomponents that make up `.card`, but it does not prescribe particular patterns. For examples of specific card builds, look to the [card patterns page](../patterns/card.md) for guidance.
+
+Also note that while many examples below use a width atomic, it's generally advisable not to do this, and to let your card width fit within an otherwise specified layout or grid.
+
+```html
+<div class="card width-300">
+	<div class="card-template">
+		<p class="card-supertitle">Course</p>
+		<a href="#" class="card-title">Learn how to create a card component</a>
+		<img
+			aria-hidden="true"
+			class="card-template-icon"
+			src="https://docs.microsoft.com/en-us/learn/achievements/review-microsoft-azure-pricing-slas-lifecycles.svg"
+		/>
+		<div class="card-template-detail">
+			<p>Additionaldetails or elements can be placed here.</p>
```

Comment:
Space between the "Additional" and "details" is missed in all recently updated spots

Reply:
whoops, added back

Reply:
correction, now they are back